packages/react-dom/src/client/setInnerHTML.js

```diff
@@ -9,25 +9,26 @@ import {Namespaces} from '../shared/DOMNamespaces';
 import createMicrosoftUnsafeLocalFunction
   from '../shared/createMicrosoftUnsafeLocalFunction';
 
-// SVG temp container for IE lacking innerHTML
-var reusableSVGContainer;
-
 /**
  * Set the innerHTML property of a node
  *
  * @param {DOMElement} node
  * @param {string} html
  * @internal
  */
-var setInnerHTML = createMicrosoftUnsafeLocalFunction(function(node, html) {
+const setInnerHTML = createMicrosoftUnsafeLocalFunction(function(node, html) {
   // IE does not have innerHTML for SVG nodes, so instead we inject the
   // new markup in a temp node and then move the child nodes across into
   // the target node
+
+  // SVG temp container for IE lacking innerHTML
+  let reusableSVGContainer;
```

Comment:
It is intentionally *outside* of the function. We don't want to allocate one per call.

Reply:
Make sense. Updated `ad3223a`